Utility/SQL/Migration.py

```diff
@@ -0,0 +1,143 @@
+#!/usr/bin/python
+# coding: utf-8
+
```

Comment:
Different formatting styles for lines (.format, '%s') are used, which makes code readability worse
Readability counts. (c) The Zen of Python by Tim Peters

Reply:
For now, this is a draft. But thanks for the advice nonetheless. I would also advise you to study C++ more thoroughly instead of python zen

Reply:
We are a team, we must help each other, thank you!